driver/compile.go

```diff
@@ -1,7 +1,6 @@
 package driver
```

Comment:
This file no longer has any compile()'s - maybe it should be renamed to something like transforms.go

Reply:
I put the `compile` function back here in c191813 ; having a separate multisource compile function made more sense before the `Run`/`Compile` combining.